CGPs/cgp-0091.md

```diff
@@ -2,10 +2,10 @@
 cgp: 91
 title: Road to COP28
 date-created: 2023-08-09
-author: 'jeanne bloch <jeannebloch@decentralized-desires.com>, '
-status: PROPOSED
+author: 'jeanne bloch <jeannebloch@decentralized-desires.com>'
+status: REJECTED
 discussions-to: 'https://forum.celo.org/t/proposal-web3-x-sdg-event-proposal-for-road-to-cop28/6225'
-governance-proposal-id: 128
+governance-proposal-id: 122
```

Comment:
This was a strange one. It looks like @juanjgiraldoc you set this to 128 here:
https://github.com/celo-org/governance/commit/7cffd4686297032b2ec0866541fc09f42522e105

But I'm not seeing any on-chain records that a 128 was ever created. At least not properly. There is record of a failed 122 that initially corresponds with this CGP